CustomAlert/Classes/CustomAlert.swift

```diff
@@ -10,29 +10,42 @@ import UIKit
 
 public protocol CustomAlertDelegate: class {
     func dismissAlert(animated: Bool)
+    func dismissAlert(animated: Bool, completion: @escaping () -> Void)
 }
 
 open class CustomAlertView: UIView {
     public weak var delegate: CustomAlertDelegate?
 }
 
-public class CustomAlert<T: UIView> {
-    let alertController: CustomAlertController
-    public let alertView: T
+public class CustomAlert {
```

Comment:
na co je to class vlastne? podla vsetkeho to nevyzera na nieco co by som chcel podedit.

Reply:
Potrebuju menit parametry v dismiss a createAlertController metodach